identity/app/model/EmailSubscriptions.scala

```diff
@@ -97,6 +97,7 @@ object EmailSubscriptions {
       frequency = "Tuesday mornings",
       listId = "3698",
       subscribedTo = subscribedListIds.exists{ x => x == "3698" },
+      signupPage = Some("https://www.theguardian.com/politics/2016/may/31/eu-referendum-morning-briefing-sign-up"),
```

Comment:
LinkTo is only relative in lcoalhost, not in CODE and not in PROD. Or use our local env variables!  Best not to hardcode this.